physionet-django/console/templates/console/console_navbar.html

```diff
@@ -136,7 +136,7 @@
       {% endif %}
 
       <!-- legal -->
-      {% if perms.project.add_dua or perms.project.add_codeofconduct or perms.project.add_license %}
+      {% if perms.project.add_dua or perms.project.add_codeofconduct or perms.project.add_license or  perms.events.add_event %}
```

Comment:
Extra whitespace here

```suggestion
      {% if perms.project.add_dua or perms.project.add_codeofconduct or perms.project.add_license or perms.events.add_event %}
```